bootique-cayenne/pom.xml

```diff
@@ -33,6 +33,10 @@
 	<name>bootique-cayenne: Cayenne Integration Bundle for Bootique</name>
 	<description>Provides Apache Cayenne integration with Bootique</description>
 
+	<properties>
```

Comment:
Cayenne "dev line"  version (4.0.x or 4.1.x) is now duplicated between cayenne, cayenne-jcache and cayenne-test.  Let's define both in the parent pom as properties to simplify tracking and upgrade